apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -70,6 +70,10 @@ export interface GenericIndexTablePresenterProps<T extends IBaseViewModel> {
 
 const allProperties = 'allProperties';
 
+// const StyledScrollableDiv = styled('div')({
```

Comment:
Is there a reason for keeping this commented out code?

Reply:
nope, gone.